net.blix.BlueMail.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.blix.BlueMail</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>Proprietary</project_license>
+  <name>BlueMail</name>
+  <developer_name>Blix Inc.</developer_name>
+  <url type="homepage">https://bluemail.me</url>
+  <summary>BlueMail Email and Calendar App</summary>
+  <description>
+    <p>BlueMail by Blix is a free, beautifully designed, universal email app, capable of managing an unlimited number of mail accounts from various providers, allowing for smart push notifications and group emailing while enabling personalization across multiple email accounts. The BlueMail app connects directly to your mail server and is the perfect replacement for your stock email app.</p>
+    <p>This is the proprietary Blix build of BlueMail, packaged into a Flatpak. This repackaging is not supported by Blix.</p>
```

Comment:
This should be changed to follow the disclaimer at the bottom of https://beta.flathub.org/apps/details/com.jetbrains.PyCharm-Professional.